### DISCUSSION TITLE
GitHub Copilot - Lesson 6

### DISCUSSION THREAD

**Discussion by itsmohammadh** [Cyfrin/foundry-full-course-cu#728] [Q&A]
hi, i can't install GitHub Copilot extension, what extension can i install instead ?

**Comment by theirrationalone** [comment]
Sorry to say, But we need to pay if we wanna use GitHub Copilot.... and if we have Paid then we can search for `GitHub.copilot` on vs-code extensions menu tab.

**Reply by shikhar229169** [reply]
But if you are a student and have a student id card, it's free for you.

**Reply by theirrationalone** [reply]
Absolutely @shikhar229169 ! 🔥🔥

**Comment by albin-george-kurian** [comment]
@itsmohammadh search for GitHub Copilot  in vscode extension search bar then you will find it but GitHub Copilot is not free..  
if you are an student you can get discount on you Student ID 



![Screenshot 2023-09-16 152709](https://github.com/Cyfrin/foundry-full-course-f23/assets/106237003/563f5516-528a-417e-bb52-aab195a82189)
